src/extension.ts

```diff
@@ -184,18 +207,37 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
         synchronize: {
             // Synchronize the setting section 'psalm' to the server (TODO: server side support)
             configurationSection: 'psalm',
-            fileEvents: vscode.workspace.createFileSystemWatcher('**/' + psalmConfigPath)
+            fileEvents: [
+                vscode.workspace.createFileSystemWatcher('**/' + psalmConfigPath),
+                // this is for when files get changed outside of vscode
+                vscode.workspace.createFileSystemWatcher('**/*.php'),
+            ]
         }
     };
     
+    let resolveStartingPromise: undefined|(() => void);
+    vscode.window.setStatusBarMessage("$(extensions) Psalm Language Server is starting ... please wait $(loading~spin)", new Promise((resolve, reject) => {
+        resolveStartingPromise = () => { resolve(); }
+    }));

```

Comment:
There is now a status on the statusbar when the language server is starting, so the end-user can see when it is available.